src/Fixer/ClassNotation/VisibilityRequiredFixer.php

```diff
@@ -176,6 +176,11 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
 
         $propertyTypeDeclarationKinds = [T_STRING, T_NS_SEPARATOR, CT::T_NULLABLE_TYPE, CT::T_ARRAY_TYPEHINT, CT::T_TYPE_ALTERNATION, CT::T_TYPE_INTERSECTION, CT::T_DISJUNCTIVE_NORMAL_FORM_TYPE_PARENTHESIS_OPEN, CT::T_DISJUNCTIVE_NORMAL_FORM_TYPE_PARENTHESIS_CLOSE];
 
+        if (\defined('T_PRIVATE_SET')) { // @TODO: drop condition when PHP 8.4+ is required
+            $propertyAsymetricType = [T_PRIVATE_SET, T_PROTECTED_SET, T_PUBLIC_SET];
+            array_push($propertyTypeDeclarationKinds, ...$propertyAsymetricType);
```

Comment:
Escaped Mutant for Mutator "ArrayItemRemoval":

Because Mutant run on PHP < 8.4.
Running Infection on 8.4 should resolve that.